src/rust/rust-proto/src/protocol/status.rs

```diff
@@ -129,7 +129,7 @@ impl From<GrpcCode> for Code {
 #[derive(thiserror::Error)]
 pub struct Status {
     /// The gRPC status code, found in the `grpc-status` header.
-    code: Code,
+    pub code: Code,
```

Comment:
i think the 'matching on a Code' is a common enough operation that making this pub makes sense

Reply:
I'm fine with it, but `pub` also means that it can be mutated. Getters are more restrictive. I get no preference.